tools/create-translation-issues.sh

```diff
@@ -133,22 +129,20 @@ Please [replace with new URLs](https://github.com/jamulussoftware/jamuluswebsite
 
 '
 
-
-
 get_languages() {
     if [[ $TYPE == app ]]; then
         if [[ ! -f src/main.cpp ]]; then
-            echo "Error: Please ensure that you are at the root of a jamulus code checkout" >/dev/stderr
+            echo "Error: Please ensure that you are at the root of a jamulus code checkout" > /dev/stderr
             exit 1
         fi
         for LANG_FILE in src/res/translation/*.ts; do
-            LANG=${LANG_FILE/*\/translation_}
-            LANG=${LANG/.ts}
+            LANG=${LANG_FILE/*\/translation_/}
+            LANG=${LANG/.ts/}
```

Comment:
What's the `/` at the end for?

Reply:
The full substitution syntax is SOURCE_VARIABLE/SEARCH/REPLACEMENT. The code in question replaces the matches by the empty string, so the last field is optional and so is the slash. `shfmt` wants to make this more explicit.

Reply:
Ah ok. Thanks.